etc/profile-a-l/armcord.profile

```diff
@@ -0,0 +1,43 @@
+# Firejail profile for armcord
+# Description: Standalone Discord client
+# This file is overwritten after every install/update
+# Persistent local customizations
+include armcord.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/ArmCord
+
+# sh is needed to allow Firefox to open links
+#include allow-bin-sh.inc
```

Comment:
Is the entry supposed to be commented?

It is uncommented in other profiles that also have the firefox part.

Reply:
Strictly speaking we can drop both lines: `nothing` down the include chain has `include disable-shell.inc`. That is different in all other profiles that have this 'open links with Firefox' support. For the latter it's a hard-dependency so to speak. I left it in (commented) on purpose. To have a reminder this is needed _if_ something changes this situation.